test-app/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstemplatepackagename/service/MessageListener.kt

```diff
@@ -2,15 +2,21 @@ package uk.gov.justice.digital.hmpps.hmppstemplatepackagename.service
 
 import com.fasterxml.jackson.databind.ObjectMapper
 import io.awspring.cloud.sqs.annotation.SqsListener
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.future.future
 import org.springframework.stereotype.Service
+import java.util.concurrent.CompletableFuture
 
 @Service
 class InboundMessageListener(private val inboundMessageService: InboundMessageService, private val objectMapper: ObjectMapper) {
 
   @SqsListener("inboundqueue", factory = "hmppsQueueContainerFactoryProxy")
-  fun processMessage(message: Message) {
+  fun processMessage(message: Message): CompletableFuture<Void> {
```

Comment:
Both `test-app` and `test-app-reactive` now have 1 synchronous and 1 asynchronous message listener